en/services/manual_control.md

```diff
@@ -0,0 +1,88 @@
+Unused axes can be disabled, and the extension axes must be explicitly enabled using bits 0 and 1 of the `enabled_extensions` field.
+
+## Mapping Axes
+
+Manual control is performed in the vehicle-frame. All axis values are normalised to the range -1000 to 1000.
+
+### Rotation-Focused Control
+
+The typical axis assignments for a thrust- and rotation-controlled vehicle (e.g. planes, multicopters) are listed below.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | pitch | forward/nose-down | backward/nose-up
+`y` | roll | right-down | left-down
+`z` | thrust | positive | negative
+`r` | yaw | counter-clockwise | clockwise
+
+### Directional Control
+
+Vehicles with direct control over vehicle translation directions (multicopters) typically use the following mappings.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | forward | forward | backward
+`y` | lateral | right | left
+`z` | vertical | up | down
+`r` | yaw | counter-clockwise | clockwise
+`s` | pitch | forward/nose-down | backward/nose-up
+`t` | roll | right-down | left-down
+
+## Mapping Buttons
+
+Button functions are vehicle-specific and determined in the vehicle firmware.
+They are generally user-configurable using firmware parameters (e.g. ArduCopter's [`BTN_FUNCn`](https://ardupilot.org/copter/docs/parameters.html#btn-func1-button-pin-1-rc-channel-function) or ArduSub's [`BTNn_FUNCTION`](https://www.ardusub.com/developers/full-parameter-list.html#btnnfunction-function-for-button)), through the [Parameter](./parameter.md) or [Extended Parameter](./parameter_ext.md) protocols.
+
+The `buttons` field is required, and corresponds to the first 16 buttons.
+
+`buttons2` is an [extension](https://mavlink.io/en/guide/define_xml_element.html#message_extensions), and corresponds to the optional second set of 16 buttons.
+
+## Future Extensions
+
+The Manual Control Protocol is by intent relatively simple and abstract, and provides a simple way of controlling the main motion of a vehicle, along with several arbitrary features that can be configured predominantly in firmware.
+This has the benefit that new vehicle types with unusual functions can be set up to operate with minimal changes required to the MAVLink protocol or existing ground control station (GCS) software.
+It also allows new GCS software to provide a simple level of control over several MAVLink-compatible vehicles.
+
+As vehicle design matures, and features increase in complexity beyond what can be reasonably controlled with 6 motion axes and a set of binary buttons states, it is recommended to implement additional targetted MAVLink commands (see the [Command Protocol](./command.md)), and handle more complex inputs in the GCS to reduce vehicle firmware complexity.
+
+## Alternatives
```

Comment:
I propose we omit this section. 

The information about the various ways of providing remote/manual control is very useful. However the way I would do that is have a separate document on RC control protocols and link to it. Then we might further have a "parent document that groups all the mechanisms for sending this kind of data to a flight stack. 

I also kind of feel that presenting RC overrides as an alternative here is a bit of a mis-use. Ideally we should promote clear integration paths that don't leverage features from other API.

Reply:
> The information about the various ways of providing remote/manual control is very useful.

Glad you think so - I definitely found it that way when writing it down, since the intended differences between the messages is something that's been unclear to me for a while.

> the way I would do that is have a separate document on RC control protocols and link to it. Then we might further have a "parent document that groups all the mechanisms for sending this kind of data to a flight stack.

Fair enough, it definitely doesn't have to be here, but would be good to at least have written down somewhere :-)

> I also kind of feel that presenting RC overrides as an alternative here is a bit of a mis-use. Ideally we should promote clear integration paths that don't leverage features from other API.

I agree with this ideal, but would note that at least in the case of ArduSub (which I'm most familiar with) they are just straight alternatives with slightly different tradeoffs, especially when sending commands via pymavlink. For some background on that, because ArduSub systems are generally tethered (due to generally no feasible means of actual radio control underwater), MAVLink messages are all passed to the autopilot via the companion computer, so there's no physical difference (values of different physical rc channels vs more structured messages, if I'm understanding correctly) and they end up as basically just different commands to similar-ish functionality. That's not to say it should be presented like that in ArduPilot docs, just saying that that's already how it's used in some cases.